ReactNativeClient/lib/TemplateUtils.js

```diff
@@ -56,7 +56,7 @@ TemplateUtils.loadTemplates = async function(filePath) {
 		});
 	}
 
-	return templates;
+	return templates.sort((a, b) => { return a.label.localeCompare(b.label); });
```

Comment:
Shouldn't we sort in a case insensitive way? (not sure if localeCompare does this)

Reply:
You're probably right, I just copied this function from the tag sorting. I'll look into a better method for sorting.

Reply:
I changed this by adding the sensitivity option.